docs/index.rst

```diff
@@ -37,11 +37,21 @@ We provide some examples to demonstrate some features of *Cheetah* and show how
     examples/optimize_speed
     examples/gradientbased
 
+Getting Started
+---------------
+
+These pages explain how to get started with *Cheetah*.
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Getting Started
+    
+    coordinate_system.md
```

Comment:
Hmm ... I'm not exactly sure about this structure with `Examples` and `Getting Started` (or whether to fix it now or clean up later).

To get started I would actually want people to go through the examples and learn about the coordinate system. I also think we will add more pages like the coordinate system one over time.

I don't like the idea of letting things go "messy" and having to fix this later, but I also wouldn't be too opposed to merge it as is.

Reply:
@cr-xu what is your opinion on this? Otherwise, I think we can merge.

Reply:
I think when we have more pages like the coordinate systems we can write a paragraph explaining the steps one need to go through to understand how cheetah works.
But for now it's clean enough and I would just merge it.